docs/upgrade-guides/GUIDELINES.mdx

```diff
@@ -0,0 +1,55 @@
+---
+draft: true
+---
+
+# How to write an upgrade guide
+
+## Guidelines
+
+### Structure
+
+The upgrade guide should be structured as follows:
+
+1. Introduction/context (no heading)
+2. Upgrade step for each breaking change (h2)
+
+### Writing
+
+Please try to be as concise as possible. The upgrade guide should be a quick reference for developers who want to upgrade their projects to the latest version. Focus on what could make the upgrading process easier for the developer: actionable items, tips, code snippets, etc.
+
+---
+
+## Example upgrade guide
+
+This migration guide describes the upgrade from version 1.0.0 to 1.1.0. Version 1.1.0 contains the following breaking changes:
+
+- The `ping` function has been removed.
+- The `nft` property was removed from the `product` object.
+
+:::caution
+
+Before you start the upgrade, make sure you to stop the server first and have a backup of your database.
+
+:::
+
+To successfully upgrade to version 1.1.0, you need to perform these steps:
+
+`## The "ping" function has been removed`
+
+In the previous versions, you would use the `ping` function to send an echo request to the Saleor API. In version 1.1.0, we removed the `ping` and introduced the alternative: `pong`. You can read more about the `pong` function in the API reference.
```

Comment:
In the real upgrade guide, we can attach the correct link to the API reference. ☝️